docs/sql-guide/functions/function-coalesce.md

```diff
@@ -0,0 +1,116 @@
+---
+title: COALESCE()
+layout: default
+parent: Functions
+grand_parent: SQL guide
+---
+
+# COALESCE() function
+
+`COALESCE()` function is used to return the first non-null value among the provided parameters.
+
+## Syntax
+
+```
+coalesce(value1, value2, ...)
+```
+
+## Arguments
+
+| Argument | Description | Length |
+|---|---|---|---|
+| `value1`,`value2`.. | Any expressions or values of same data type. The function returns the first non-null value among them. | 1 |
```

Comment:
You've said value1, value2 are expressions or values of the same data type.
Does this mean users can insert *actual* expressions or is this just phrasing?
And in the examples there's quoted strings that aren't clear to me from the syntax.
The length doesn't seem to correspond to the values returned in the examples?